docs/fields/overview.mdx

```diff
@@ -138,3 +139,27 @@ The `condition` function should return a boolean that will control if the field
 ### Custom components
 
 All Payload fields support the ability to swap in your own React components with ease. For more information, including examples, [click here](/docs/admin/components#fields).
+
+### Description
+
+You can simply provide a string that will show by the field, but there are use cases where you may want to create some dynamic feedback. By using a function for the `description` property you can provide rich feedback in realtime the user interacts with the form.
+
+The `description` can be assigned a function can wth the parameter `{ value }` and can return the `string` you wish to display.
```

Comment:
Does this _**need**_ to return a string? Shouldn't it still work if you returned a React component?